doc/overview.rst

```diff
@@ -1,33 +1,32 @@
 Overview
 =========
 
-SaltProc couples directly with the Monte Carlo neutron transport code Serpent 2
-and enables sophisticated, multi-component online fuel reprocessing system
+SaltProc couples directly with a variety of Monte Carlo neutron transport codes with depletion capabilities and enables sophisticated, multi-component online fuel reprocessing system
 modeling. SaltProc is the first open-source tool for liquid-fueled MSR
 depletion simulation with the following capabilities:
 
 - neutron poison removal with user-defined efficiency,
 - make-up mass loss by fresh fuel injection,
-- reactivity control by adjusting fuel feed rate or geometry change,
+- reactivity control by adjusting fuel feed rate or geometry,
```

Comment:
Does this point mean just fuel geometry? Or problem geometry?

Reply:
ok can you edit it to clarify that? The way I'm reading this I interpret it as fuel geometry